extension/src/popup/components/manageAssets/ManageAssetRows/index.tsx

```diff
@@ -205,64 +198,10 @@ export const ManageAssetRows = ({
       dispatch(resetSubmission());
       navigateTo(ROUTES.account);
     }
-  }, [submitStatus, assetSubmitting, dispatch]);
```

Comment:
I removed `assetSubmitting` because it's not used. We can put it back if it needs to trigger something.

Reply:
oh weird, I'm surprised that this didn't trigger a warning before 🤔